rdock-utils/rdock_utils/sdsort/parser.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class SDSortConfig:
+    sorting_field: str
+    reverse_sort: bool
+    numeric_sort: bool
+    fast_mode: bool
+    group_key: str
+    files: list[str] | None
+
+
+def get_parser() -> argparse.ArgumentParser:
+    description = """Sorts SD records by a specified data field.
+    Notes:
+    - "_REC" (record #) is provided as a pseudo-data field.
+    - If no SD file list is provided, the script reads from standard input.
+    - Output is directed to standard output.
+    - Fast mode can be safely used for partial sorting of large SD files of raw docking hits without encountering memory issues.
```

Comment:
```suggestion
    - Fast mode can be safely used for partial sorting of large SD files of raw docking hits without encountering memory issues. It will sort together consecutive molecules with the same value for a grouping key (the title, by default) instead of sorting all provided molecules together.
```